app/javascript/components/project-submissions/components/__tests__/submission-title.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import SubmissionTitle from '../submission-title';
+
+const submission = {
+  lesson_path: 'test_path',
+  lesson_title: 'Test Title',
+  user_name: 'TestUser',
+};
+
+describe('Dashboard view true', () => {
+  it('shows the lesson title in dashboard view', () => {
+    render(<SubmissionTitle submission={submission} isDashboardView={true} />);
```

Comment:
Write out in multi-line, since you are passing in more than 1 props. See other tests for example.